docs/source/download.rst

```diff
@@ -1,64 +1,94 @@
+===========================
+Download and Install Devito
+===========================
+
+There are three main approaches to installing Devito.
+
+For those looking for a quick “out-of-the-box” way to try Devito, we recommend
+the route of `Docker installation`_.
+For those looking to use Devito alongside other packages as part of a project, we support
+`pip installation`_. If you are looking to develop for Devito, you might benefit from
+following the `conda installation`_  that includes all the bells and whistles we
 recommend when developing for Devito.
 
+
 Docker installation
 -------------------
+
 You can install and run Devito via Docker_:
 
 .. _Docker: https://www.docker.com/  
 
+Docker for CPUs
+```````````````
+
 .. code-block:: shell
 
-   # get the code
-   git clone https://github.com/devitocodes/devito.git
-   cd devito
+   # NOTE (admin-sudo access may be needed) for some steps
+
+   # 1. Pull Devito image
+   docker pull devitocodes/devito:cpu-latest
 
-   # 1. run the tests
-   docker-compose run devito /tests
+   # 2. (Optional but recommended) Run the tests
+   docker run --rm --name testrun 'devitocodes/devito:cpu-latest' pytest tests/test_operator.py
 
-   # 2. start a jupyter notebook server on port 8888
-   docker-compose up devito
+   # 3. Start a bash shell with Devito
+   docker run --rm -it devitocodes/devito:cpu-latest /bin/bash
 
-   # 3. start a bash shell with devito
-   docker-compose run devito /bin/bash
+   # 4. Start a Jupyter notebook server on port 8888
+   docker run --rm -it -p 8888:8888 devitocodes/devito:cpu-latest
 
-The three sample commands above have only been included as illustrations of typical
+These sample commands above have only been included as illustrations of typical
 uses of Devito inside a container. They are not required to be run in that order.
+Please also review these commands within the dockerfile itself:
+`Dockerfile <https://github.com/devitocodes/devito/blob/master/docker/Dockerfile>`_
 
-1. Command 1 above runs the unit tests included with Devito to check whether the
-installation succeeded. This is not necessary but a handy first thing to run. As
-long as the first few tests are passing, it is possible to press `Ctrl+C` to stop
-the testing and continue.
 
-2. Command 2 starts a Jupyter_ notebook server inside the
-docker container and forwards the port to `http://localhost:8888`.
-After running this command, you can copy-paste the complete URL from the terminal window where
-the command was run - to a browser to open a jupyter session to try out the included
-tutorials. Alternatively, you may simply point your browser to `http://localhost:8888`
-and, if prompted for a password, copy-paste the authentication token from the command
-window. Once successfully in the Jupyter notebook session, proceed to run the tutorials
-provided in the `examples` folder or create your own notebooks. 
+Docker for GPUs
+```````````````
+
```

Comment:
Add something in the line of

The GPU image only differs from the CPU image in term of compilers but contains the same installation and usages of Devito .....